### DISCUSSION TITLE
Single install of Star Wars: Jedi Fallen Order showing as duplicate for both Jedi: Fallen Order and Jedi: Survivor?

### DISCUSSION THREAD

**Discussion by Garfieldwxg55** [mtkennerly/ludusavi#433] [Q&A] [answered]
Hi, so I have a single install of Star Wars Jedi: Fallen Order installed on my PC through EA App. However, I noticed that Ludisavi is sensing that my GameUserSettings.ini file is duplicated. Further, it shows as being duplicated for both Jedi: Fallen Order, and Jedi: Survivor, the latter of which I own but have never played (I'm not sure if I have even ever installed it).

I've added a screenshot of the problem here:
<img width="756" alt="Ludisavi Issue" src="https://github.com/user-attachments/assets/e6e18e12-ae60-42d6-8d3c-7aa5718a4686" />

Does anyone know what might be causing this issue? And if so, how I might be able to fix it?

Many thanks!

**Comment by mtkennerly** [comment]
Hi! According to PCGamingWiki, these two games both use the same config file, so Ludusavi can't know which one it's technically from:

* https://www.pcgamingwiki.com/wiki/Star_Wars_Jedi:_Fallen_Order#Game_data
* https://www.pcgamingwiki.com/wiki/Star_Wars_Jedi:_Survivor#Game_data

There are a couple of options you can use to resolve the duplicates:

* Uncheck `Star Wars Jedi: Survivor` (either the whole game or just the conflicting file). That will clear the duplicates for both games (the "duplicates" badge will fade), but will leave `Star Wars Jedi: Survivor` in the scan list.
* Make a custom game entry called `Star Wars Jedi: Survivor`, mark it as "override" (which is the default), and then remove this path: `<winLocalAppData>/SwGame/Saved/Config/WindowsNoEditor/GameUserSettings.ini` . This will keep Ludusavi from scanning that file for that game at all, and since that's the only file being found, the game won't show up in your scan results at all (unless you play it eventually and it gets other save files on your system).

**Reply by Garfieldwxg55** [reply]
Hi, thanks for the reply!

I will probably use the first option, however, I was curious what I might do in the future when I decide to play Star Wars Jedi: Survivor, to ensure that both games have a backup (even if it ultimately ends up being the same file).

Thanks again for your help!

**Reply by mtkennerly** [reply, marked as answer]
It's up to your preference, but if you may play Survivor eventually, then personally I would just leave the duplicate enabled. That way, you don't have to worry about one of the games having a newer/older backup of that file - they would both always have the latest copy. I'd rather make sure I back up as much as possible, and I don't mind having a few duplicates or unnecessary backups in exchange, but that's just my opinion  :)

**Reply by Garfieldwxg55** [reply]
In that case, I'll just leave the duplicate enabled, thanks for your help!
